docs/tutorials/06-batch-infer.ipynb

```diff
@@ -701,7 +701,10 @@
     "\n",
     "Finally, perform the batch prediction by passing in your model and datasets. In addition, you can trigger the drift analysis batch job on the provided dataset by passing `\"trigger_monitoring_job\": True`. \n",
```

Comment:
`trigger_monitoring_job` is deprecated:
```suggestion
    "Finally, perform the batch prediction by passing in your model and datasets.\n",
```
Could you please remove the other two occurrences of it in `docs/deployment/batch_inference.ipynb`?